src/test/java/bio/terra/service/dataset/flight/inheritsteward/AdjustStewardMembersStepTest.java

```diff
@@ -44,33 +44,37 @@ interface DoOrUndo {
   private void verifyAdjustMembers(DoOrUndo doOrUndo, boolean inheritSteward) throws Exception {
     var snapshots =
         List.of(new Snapshot().id(UUID.randomUUID()), new Snapshot().id(UUID.randomUUID()));
-    var custodianUser = "user";
+    List<String> datasetPolicyEmails = new ArrayList<>(List.of("custodianEmail", "stewardEmail"));
```

Comment:
Using `Arrays.asList()` also works here
```suggestion
    List<String> datasetPolicyEmails = Arrays.asList("custodianEmail", "stewardEmail");
```